data/keys

```diff
@@ -39,20 +39,27 @@ Global {
 	KeyPress = "Ctrl Mod1 Shift Right" { Actions = "SendToWorkspace Prev; GoToWorkspace Prev" }
 	KeyPress = "Ctrl Mod1 Shift Up" { Actions = "SendToWorkspace NextV; GoToWorkspace NextV" }
 	KeyPress = "Ctrl Mod1 Shift Down" { Actions = "SendToWorkspace PrevV; GoToWorkspace PrevV" }
-	KeyPress = "Mod4 F1" { Actions = "SendToWorkspace 1" }
-	KeyPress = "Mod4 F2" { Actions = "SendToWorkspace 2" }
-	KeyPress = "Mod4 F3" { Actions = "SendToWorkspace 3" }
-	KeyPress = "Mod4 F4" { Actions = "SendToWorkspace 4" }
-	KeyPress = "Mod4 F5" { Actions = "SendToWorkspace 5" }
-	KeyPress = "Mod4 F6" { Actions = "SendToWorkspace 6" }
-	KeyPress = "Mod4 F7" { Actions = "SendToWorkspace 7" }
-	KeyPress = "Mod4 F8" { Actions = "SendToWorkspace 8" }
-	KeyPress = "Mod4 F9" { Actions = "SendToWorkspace 9" }
+	KeyPress = "Mod4 Shift 1" { Actions = "SendToWorkspace 1" }
```

Comment:
Using Shift here and the number keys like for the GoToWorkspace bindings seems more consistent to me.